mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -35,6 +34,8 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const natPunchingMaxTTL = 10
```

Comment:
I suggest to change it to more clear name like `natPunchingPortsLen` etc. In this context TTL is a bit confusing with IP packets TTL.

Reply:
The number of ports is needed to test all TTL values up to the `natPunchingMaxTTL`.